src/EA.Iws.Web/Areas/AdminExportNotificationMovements/Controllers/CaptureMovementController.cs

```diff
@@ -201,5 +207,15 @@ public ActionResult Cancelled(Guid id)
             var summary = await mediator.SendAsync(new GetInternalMovementSummary(id));
             model.SetSummaryData(summary);
         }
+
+        private bool CanShowEditLink()
```

Comment:
This method should be made `async` so you can await on the `AuthorizeActivity` call.

`private async Task<bool> CanShowEditLink()`